app/projects/route.js

```diff
@@ -0,0 +1,19 @@
+import { reads } from "@ember/object/computed";
+import Route from "@ember/routing/route";
+import { inject as service } from "@ember/service";
+
+export default Route.extend({
+  session: service(),
+  user: reads("session.data.user"),
+
+  setupController(controller, ...args) {
+    this._super(...args);
+
+    if (this.user.isSuperuser || this.user.isReviewer) {
+      controller.set("hasAccess", true);
+      controller.fetchProjectsOfUser.perform();
+    } else {
+      controller.set("hasAccess", false);
+    }
+  }
+});
```

Comment:
Why did you use a route?

Reply:
@fkm fixed used `init()` inside the controller instead